### PR TITLE
Ad Shelly as a velero-plugin admin

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/orgs.yaml [hunk 1]
```diff
@@ -584,6 +584,7 @@ orgs:
           - tomob
           - aglitke
           - brybacki
+          - shellyka13
```
Comment (akalenyu) [inline github/ci/prow-deploy/files/orgs.yaml:587]: just double-checking if case matters here, because it's spelled @ShellyKa13

Comment (akalenyu) [reply github/ci/prow-deploy/files/orgs.yaml:587]: right, I do see some usernames in this file that are different, but that doesn't matter I guess

**Review by brianmcarey — APPROVED** [review]
/approve

**Pull request merged**